contracts/CourtRoom.sol

```diff
@@ -146,16 +146,17 @@ contract SwearGame is Owned {
     if (decision == true){
 	    caseCompensated = compensate(plaintiff);
 			caseContract.resolveClaim(_id);
+      leaveGame(plaintiff);
```

Comment:
In general this is for a possible situation where there is not enough deposit for all already registered players.
Any how ,this issue/question should be raised again on pr#5 ..it is not solved yet .